prqlc/prqlc/src/sql/pq/context.rs

```diff
@@ -169,7 +167,6 @@ impl AnchorContext {
 
         let original_cids = table_ref.columns.iter().map(|(_, c)| *c).collect();
         let relation_instance = RelationInstance {
-            riid,
```

Comment:
riid (relational instance id) is not really needed in `RelationInstance`, because these structs are stored in `HashMap<RIId, RelationInstance>` anyway.

So this edit is ok.